### DISCUSSION TITLE
Icon naming

### DISCUSSION THREAD

**Discussion by wokija** [zixpo/candybar#43] [Q&A] [answered]
I was wondering how icons filenames work exactly since I can't seem to find infos on Candybar Wiki unfortunately. For instance, I though double underscore would show next letter as capital in app's icon preview (you__tube -> YouTube) or would I need to edit _bodyboard_icon_name_replacer.xml_?

Same with lowercase at beginning of filename. How would you name your file to show as let's say "iMessage"?

**Comment by sarsamurmu** [comment, marked as answer]
CandyBar supports setting different name to show in the dashboard icon viewer. But you won't have to edit the original file name. You can just use the `name` attribute on any `item` of `drawable.xml`.

Example -
```xml
<item drawable="youtube" name="YouTube" />
```
In this case `youtube` icon would show up as `YouTube`

I would update the wiki later to include this.

**Reply by wokija** [reply]
Forgot to reply to thank you for your fast answer. I tested it and it even support ampersand (using `&amp;`)
